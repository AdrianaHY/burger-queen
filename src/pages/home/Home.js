import React, { Component } from 'react';

import whiteLogo from '../../assets/white_logo.png';

import './Home.scss';

class Home extends Component {
	render() {
		const { handlerMenuPage, handlerKitchen } = this.props;
		return (
			<div className="home">
				<img src={whiteLogo} alt="logo"/>
				<h1 className="home-title">Who is using this app?</h1>
				<div className="button-container">
					<button
						onClick={handlerMenuPage}
						className="home-button"
					>
						waiter/waitress
					</button>
					<button
						onClick={handlerKitchen}
						className="home-button"
					>
						kitchen
					</button>
				</div>
			</div>
		)
	}
}

export default Home;