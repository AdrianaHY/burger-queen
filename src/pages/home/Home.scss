@import '../../styles/_variables.scss';

.home {
	background: $purple;
	height: 100vh;
	color: $white;
	display: flex;
	flex-direction: column;
	justify-content: center;
	align-items: center;

	.home-title {
		font-size: $title;
	}

	.button-container {
		width: 100vw;
	}

	.home-button {
		color: $white;
		font-size: $fontSizeHuge;
		border: 2px solid $white;
		background: transparent;
		border-radius: 5%;
		padding: $paddingMedium $paddingLarge;
		margin: $paddingMedium;
	}
}