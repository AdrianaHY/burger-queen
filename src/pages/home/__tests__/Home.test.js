import React from 'react';
import { shallow } from 'enzyme';
import Home from '../Home';

describe('pages | home | Home', () => {
	it('Renders the page', () => {
		const props = {
			handlerMenuPage: jest.fn(),
			handlerKitchen: jest.fn()
		};

		const component = shallow( <Home {...props} />)
		expect(component).toMatchSnapshot();
	});
});