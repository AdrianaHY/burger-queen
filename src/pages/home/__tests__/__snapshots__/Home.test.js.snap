// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`pages | home | Home Renders the page 1`] = `ShallowWrapper {}`;
